refinery/tool_manager/tests.py

```diff
@@ -267,9 +267,12 @@ def setUp(self):
             self.django_docker_cleanup_wait_time
         )
 
-    def load_visualizations(self):
+    def load_visualizations(self, visualizations=None):
         # TODO: More mocking, so Docker image is not downloaded
-        visualizations = ["{}/visualizations/igv.json".format(TEST_DATA_PATH)]
+        if visualizations is None:
+            visualizations = [
+                "{}/visualizations/igv.json".format(TEST_DATA_PATH)
+            ]
```

Comment:
Maybe move this into the signature? So it's clear just looking at that what the default is.